source/Calamari.Shared/Commands/Support/ConsoleFormatter.cs

```diff
@@ -14,6 +15,11 @@ public static int PrintError(Exception ex)
                 Log.Error(ex.Message);
                 return 1;
             }
+            if (ex is RecursiveDefinitionException)
+            {
+                //dont log these - they have already been logged earlier
+                return 101;
```

Comment:
New return code... will this mean some assent changes in OD Server E2E tests req'd?

Reply:
I doubt it...